recipes/tutanota/package.json

```diff
@@ -1,9 +1,13 @@
 {
   "id": "tutanota",
-  "name": "Tutanota",
-  "version": "1.3.0",
+  "name": "Tuta",
+  "version": "1.4.0",
+  "description": "Tuta Mail",
+  "aliases": [
+    "Tutanota"
```

Comment:
If the recipe name is still `tutanota` (and this is valid/necessary to be able to upgrade automatically for those users who have already configured this in their installation of Ferdium), then the alias can be `tuta`. Having the alias as the same name as the recipe is redundant.

Reply:
If your intention is for the partial string match to kick in and match on "tuta," then the alias is unnecessary

Reply:
I think the alias is useful in case someone types out "Tutanota" in full or copy-pastes the name.
Or even just as a reference to the old name shown on the "Available services" screen.

We did the same thing here:
https://github.com/ferdium/ferdium-recipes/pull/423/files

Reply:
> If your intention is for the partial string match to kick in and match on "tuta," then the alias is unnecessary

I tried that, but it didn't work.

Maybe someone can help. How can I debug this script. However in the end this isn't nessessary to migrate credentials, just manual login and done.